contrib/build_rpm.sh

```diff
@@ -28,34 +28,30 @@ declare -a PKGS=(device-mapper-devel \
                 go-compilers-golang-compiler \
                 )
 
-if [ $pkg_manager == "*dnf" ]; then
+if [[ $pkg_manager == *dnf ]]; then
     PKGS+=(python3-devel \
         python3-varlink \
         )
     # btrfs-progs-devel is not available in CentOS/RHEL-8
     if ! grep -i -q 'Red Hat\|CentOS' /etc/redhat-release; then
         PKGS+=(btrfs-progs-devel)
     fi
-
-fi
-
-# golang-github-cpuguy83-go-md2man is needed for building man pages
-# It is not available by default in CentOS 8 making it optional
-if [ -z "$extra_arg" ]; then
-    PKGS+=(golang-github-cpuguy83-go-md2man)
+    # disable doc until go-md2man rpm becomes available
+    # disable debug to avoid error: Empty %files file ~/rpmbuild/BUILD/libpod-.../debugsourcefiles.list
+    export extra_arg="--without doc --without debug"
+else
+    if ! grep -i -q 'Red Hat\|CentOS' /etc/redhat-release; then
+        PKGS+=(golang-github-cpuguy83-go-md2man)
+    fi
 fi
 
 echo ${PKGS[*]}
 sudo $pkg_manager install -y ${PKGS[*]}
 
 make -f .copr/Makefile
-rpmbuild --rebuild ${extra_arg:-""} podman-*.src.rpm
-
-# Test to make sure the install of the binary works
-sudo $pkg_manager -y install ~/rpmbuild/RPMS/x86_64/podman-*.x86_64.rpm
-
-
-# If we built python/varlink packages, we should test their installs too
-if [ $pkg_manager == "*dnf" ]; then
-    sudo $pkg_manager -y install ~/rpmbuild/RPMS/noarch/python*
+# workaround for https://github.com/containers/libpod/issues/4627
+if [ -d ~/rpmbuild/BUILD ]; then
```

Comment:
rpmbuild dir structure is created by rpmdev-setuptree command. if the dir is not there it will create it and if dir is there, it will not do any thing and owning that particular dir is not needed

Reply:
I can assure you that this was required for building rpms locally. We cannot fully control in which order the user is calling all the targets in the makefile and it happened to me multiple times to get files that we could not remove there. Removal is needed as part of the rebuild.

This additon does not impact CI as we have a stable and predictable way of calling it, and we also do not have any idempotence tests. Still, that is needed in order to be able run `make packages` multiple times without failing.